spec/schemas/_common.query_dsl.yaml

```diff
@@ -214,6 +214,9 @@ components:
           maxProperties: 1
         span_within:
           $ref: '#/components/schemas/SpanWithinQuery'
+        template:
+          description: Returns documents using a search template with placeholder values.
```

Comment:
search template is another feature, let's rephrase this sentence to be more clear.  

```
Return documents using a template query contains placeholders that are resolved by search request processors during query execution. 
```